README.md

```diff
@@ -208,16 +283,16 @@ The OPTIONAL `fct` field contains arbitrary facts and proofs of knowledge. The e
 ]
 ```
 
-### 3.2.5 Attenuation Scope
+### 3.2.5 Attenuations
 
-The attenuation scope (i.e. UCAN output, or "caveats") MUST be an array of heterogeneous access scopes (defined below). This array MAY be empty.
+The attenuations (i.e. UCAN output, or "caveats") MUST be an array of heterogeneous capabilities (defined below). This array MAY be empty.
 
 This array MUST contain some or none of the following:
 1. A strict subset (attenuation) of the witnesses
 2. Witnesses originated by the `iss` DID (i.e. by parenthood)
 3. Witnesses composed from other witnesses (see rights amplification)
```

Comment:
"Witness" in this context seems weird to me. I thought witness is the stuff inside the `prf` array. (I.e. other encoded UCANs).
I'd call them "Capabilities originated by the `iss` DID (i.e. by parenthood)" and "Capabilities composed from other capabilities in the witnesses (see rights amplification)".

Reply:
@matheus23 better?

Reply:
https://github.com/ucan-wg/spec/pull/20/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R290-R293